### PR TITLE
added netty package to currency automation

### PR DESCRIPTION
**Pull request by mohitp333**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@mohitp333 please check the travis build failure

**Comment by seth-priya** [comment]
@nikhil-kalbande

File: n/netty/netty-4.1.84.Final_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+git checkout $PACKAGE_VERSION
+
+git branch
+
+#Build and test package
+if ! mvn install ; then
+    echo "------------------$PACKAGE_NAME::install_fails-------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Fail|  install_fails"
+    exit 2
```
Comment (nikhil-kalbande) [inline n/netty/netty-4.1.84.Final_ubi_8.5.sh:42]: @mohitp333  please make this exit 1

File: n/netty/netty-4.1.84.Final_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Package       : netty
+# Version       : netty-4.1.79.Final
```
Comment (nikhil-kalbande) [inline n/netty/netty-4.1.84.Final_ubi_8.5.sh:5]: @mohitp333 correct this version to version you tested/added script

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**